src/components/layout/ToolTip.tsx

```diff
@@ -0,0 +1,15 @@
+import Tooltip from '@mui/material/Tooltip';
+import { HelpOutline } from '@mui/icons-material';
+export type TooltipProps = {
+  title: string;
+};
+
+const CustomTooltip = ({ title, ...props }: TooltipProps) => {
+  return (
+    <Tooltip arrow={true} placement='top' title={title} {...props}>
+      <HelpOutline />
+    </Tooltip>
+  );
+};
```

Comment:
Ta inn TooltipProps fra `@mui` slik at vi kan alle Tooltip sine props

```suggestion
import { Tooltip as MuiTooltip, TooltipProps as MuiTooltipProps } from '@mui/material/Tooltip';
import { HelpOutline } from '@mui/icons-material';

export type TooltipProps = MuiTooltipProps;

const Tooltip= (props: MuiTooltipProps) =>  (
    <MuiTooltip arrow={true} placement='top' {...props}>
      <HelpOutline />
    </MuiTooltip>
  );
```